ecl/hql/hqlfield.cpp

```diff
@@ -0,0 +1,25 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+#include "jliball.hpp"
+#include "hql.hpp"
+#include "eclrtl.hpp"
+
+#include "platform.h"
+#include "jlib.hpp"
+#include "hqlexpr.hpp"
+#include "hqlfield.hpp"
+
```

Comment:
Subsequent PRs based on this branch will fill it in.  I would prefer to merge as is, rather than remove them, but I can remove if @richardkchapman prefers.